test/AsDispatcher/RegisterNftAsDispatcher.ts

```diff
@@ -0,0 +1,70 @@
+import {expect} from "chai";
+import {ethers} from "hardhat";
+import {deploySystem, TEST_SALE_CREATOR_BP} from "../Scripts/setup";
+import {BigNumber} from "ethers";
+import {deriveTransformId} from "../Scripts/derivedParams";
+
+describe("MakerRegistrar RegisterNftAsDispatcher", function () {
```

Comment:
Might want to do a negative test here and try to register before adding the dispatcher.

Reply:
Done!